about.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About page</title>
+ <link rel="stylesheet" href="about.css">
+ <script src="https://kit.fontawesome.com/2849147338.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <header>
```

Comment:
- [ ] You added a` black navbar` to only home page but if you take a look at the below screenshot of the original design you can see it is in the about page as well. Kindly add it as the previous reviewer requested👇 
- [ ] Also, in your about page there is still a `home` link. Kindly remove it instead you can use your logo as a link to your home page. 👇 
- Original design :
![Screenshot (275)](https://user-images.githubusercontent.com/67087664/175492792-a3609acf-a42e-4046-a8c6-f3422b98c58d.png)
- Your design :
![Screenshot (276)](https://user-images.githubusercontent.com/67087664/175493768-47475b05-8818-45b5-9fc6-3f8f1ba12998.png)